src/Domain/Commit.php

```diff
@@ -97,4 +98,14 @@ public function comment(): string
     {
         return $this->comment;
     }
+
+    public function jsonSerialize(): array
```

Comment:
I would add more specific phpdoc for it.

```suggestion
    
    /**
     * @return array{hash: string, author: string, comment: string, commitedAt: string}
     */
    public function jsonSerialize(): array
```